x-pack/plugins/cloud_security_posture/public/pages/vulnerability_dashboard/vulnerability_table_panel.tsx

```diff
@@ -37,30 +38,38 @@ export const VulnerabilityTablePanel = <T extends {}>({
 }: VulnerabilityDashboardTableProps<T>) => {
   const { euiTheme } = useEuiTheme();
   const { title } = vulnerabilityDashboardTableContent[tableType];
-  const tooltipText = i18n.translate('xpack.csp.vulnerabilityTable.panel.tooltipText', {
-    defaultMessage: 'Explore in Findings',
-  });
 
   return (
     <ChartPanel title={title}>
-      <EuiBasicTable
-        className="vulnerabilityTable"
-        css={css`
-          & .euiTableHeaderCell {
-            border-bottom: ${euiTheme.border.width.thick} solid ${euiTheme.colors.darkestShade};
-          }
-        `}
-        items={items}
-        columns={columns}
-      />
-      <EuiSpacer size="s" />
-      <EuiToolTip position="top" content={tooltipText}>
-        <EuiButtonEmpty iconType="search" onClick={onViewVulnerabilitiesClick} size="s">
-          {i18n.translate('xpack.csp.vulnerabilityTable.panel.buttonText', {
-            defaultMessage: 'View all vulnerabilities',
-          })}
-        </EuiButtonEmpty>
-      </EuiToolTip>
+      <EuiFlexGroup direction="column" justify-content="space-between">
+        <EuiFlexItem>
+          <EuiBasicTable
+            className="vulnerabilityTable"
+            css={css`
+              & .euiTableHeaderCell {
+                border-bottom: ${euiTheme.border.width.thick} solid ${euiTheme.colors.darkestShade};
+              }
+            `}
+            items={items}
+            columns={columns}
+          />
+          <EuiSpacer size="s" />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButtonEmpty
+            css={css`
+              width: 188px;
```

Comment:
`fit-content` might be suitable here

![image](https://github.com/elastic/kibana/assets/51442161/8dcb6f12-b6aa-4fd6-a9c9-aa7702befa93)